src/components/presentational/Title.js

```diff
@@ -0,0 +1,12 @@
+import React from "react";
+import './Title.css';
+
+export default function Title({ titleText, contentName, className }) {
+    return (
+        <h2 className={`${className}`}>
+            <a name={contentName} href={`#${contentName}`} className="content-title">
+                {titleText}
+            </a>
+        </h2>
```

Comment:
я бы сделал вот такой API

```jsx
<Title id="ipsum">
  Lorem
</Title>
```

```suggestion
        <h2 id={id} className={className}>
            <a href={`#${id}`} className="content-title">
                {children}
            </a>
        </h2>
```

- `titleText` -> `children` – т.к. компонент называется `Title`, то префикс `title` не нужен, а вместо параметра `text` можно сразу использовать `children` (базовое св-во у React-элементов)
- `contentName` -> `id` – при этом атрибут падает в `<h2>`, у `<a>` атрибут `name` нам уже не нужен

> [!NOTE]
>
> `name`, как якорь, с **HTML 5** уже устарел
> 
> Теперь его используют только для элементов форм по типу `<input>`

> [!NOTE]
> 
> Отдельно отмечу, что поправил лишнюю интерполяцию, видимо забыла убрать
> ```diff
> - className={`${className}`}
> + className={className} 
> ```